### PR TITLE
Fix release version detection

### PR DESCRIPTION
**Pull request by TheAssassin**
Should fix build issues like in #721.

### CODE REVIEW

File: .github/workflows/github-build.yml [hunk 1]
```diff
@@ -31,6 +31,11 @@ jobs:
           submodules: "recursive"
           fetch-depth: 0
 
+      # continuous tag confuses git describe --tags, happens here specifically due to us fetching the entirety of the repository
+      - name: Delete continuous tag
+        run: |
+          git tag | grep '^continuous$' && git tag -d continuous
```
Comment (jnweiger) [inline .github/workflows/github-build.yml:37]:
```suggestion
          git tag -d continuous 2> /dev/null
```

Save some cpu cylces?

Comment (TheAssassin) [reply .github/workflows/github-build.yml:37]: It'll fail if the tag doesn't exist with return code 1, causing the step to fail. You'd have to `|| true` then. It won't really speed it up either, I guess. Also, allowing the output to end up in the log is handy for debugging.

Comment (TheAssassin) [reply .github/workflows/github-build.yml:37]: You could do the following:

```suggestion
          git tag -d continuous || true
```

**Comment by mgmax** [comment]
Thanks for the improvement. I'm not sure if this really solves the root cause. We will probably have the same issue if someone clones and builds locally. Looks like GitHub releases is only intended for building "release" versions and not every single commit?

**Comment by TheAssassin** [comment]
People are indeed affected locally as well. This was specifically meant to "fix" the problem on the CI for now. I know that @jnweiger has had this problem locally once already.

**Pull request merged**